client/src/index.css

```diff
@@ -29,7 +29,7 @@ code {
     monospace;
 }
 
-/* .landingPage {
+.landingPage {
```

Comment:
Why were these uncommented?

Reply:
Literall no clue, if you look at my previous commit, I pulled down development last night and spent all day fixing changes that someone had previously approved. That has no relation to my component and I'm way to lazy to actually mess with that.